p2p/test/p2ptest.go

```diff
@@ -0,0 +1,75 @@
+package p2ptest
+
+import (
+	"context"
```

Comment:
So this is super nitpickie, but I prefer separate three categories of imports with a blank line, in the form
```golang
import (
  [standard library packages]

  [third party packages]
  
  [gx packages]
)
```
I find it makes it easier to read what a file is importing this way, no need to change this for this PR, we can catch it later on, just wanted to mention.